redis/src/cluster_client.rs

```diff
@@ -238,6 +242,22 @@ impl ClusterClientBuilder {
         self
     }
 
+    /// Enables timing out on slow connection time.
+    ///
+    /// If enabled, the cluster will only wait the given time on each connection attempt to each node.
+    pub fn connection_timeout(mut self, connection_timeout: Duration) -> ClusterClientBuilder {
+        self.builder_params.connection_timeout = Some(connection_timeout);
+        self
+    }
+
+    /// Enables timing out on slow responses.
+    ///
+    /// If enabled, the cluster will only wait the given time to each response from each node.
+    pub fn response_timeout(mut self, connection_timeout: Duration) -> ClusterClientBuilder {
```

Comment:
The variable name here is also `connection_timeout`

Reply:
I agree that it should be renamed (or just call the var `timeout` in both this and the above method)